_docs/style/text-conventions/capitalization.md

```diff
@@ -0,0 +1,134 @@
+
+The following table includes examples for title-style capitalization:
+
+{% figure [caption:"Title-style capitalization examples"] [class:"image bg-light border"] %}
+| <hx-icon type="checkmark" class="good-idea"></hx-icon> Use | <hx-icon type="times" class="bad-idea"></hx-icon> Don't Use  |
+| --- | --- |
+| Restore Defaults | Restore defaults |
+| Allocate to Project | Allocate to project |
+| Show Add-on in Menu | Show Add-On in Menu |
+{: class="hxTable"}
+{% endfigure %}
+
+The following elements use title-style capitalization:
+
+- Button labels
+- Chart titles
+- Checkbox and radio button field names
+- Icon labels
+- Links used for an action or navigation
+- Menu items (both menu bar and context menus)
+- Menu titles
+- Modal titles
+- Page titles
+- Section headings
+- Step titles
+- Table column headers
+- Tab titles
+- Table titles
+- Toolbar buttons
+- Window titles
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## Sentence-style capitalization
+
+For sentence-style capitalization, most words are lowercase, with the following exceptions:
+
+- Capitalize the first letter of the first word of the text element.
+- Capitalize the first letter of any other words that you would normally capitalize, such as proper nouns.
+
+The following table shows examples for sentence-style capitalization:
+
+{% figure [caption:"Sentence-style capitalization examples"] [class:"image bg-light border"] %}
+| <hx-icon type="checkmark" class="good-idea"></hx-icon> Use | <hx-icon type="times" class="bad-idea"></hx-icon> Don't Use  |
+| --- | --- |
+| Maximum number of downloads | Maximum Number of Downloads |
+| Allocate to project | Allocate to Project |
+{: class="hxTable"}
+{% endfigure %}
+
+The following elements use sentence-style capitalization:
+
+- Checkbox and radio button help text
+- Field help text
+- Field names
+- File names
+- Headings for groups of radio buttons or checkboxes
+- Hover help text
+- Input hints
+- List box, drop-down list, and combination box entries
+- List box labels
+- Messages (information, warning, and error)
+- Option labels
+- Page instructions or descriptions
+- Progress bar labels
+- Status bar text
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## All-caps capitalization
+
+In rare cases, all-caps capitalization might be used to visually distinguish a secondary text element, such as, **GENERAL** as a menu heading.
+
+The following elements use all-caps capitalization:
+
+- Header 5
+- Table column headers
+- Alert bar messages
```

Comment:
it's not the whole message. just the message um... "header" maybe? check it: http://helix.rax.io/components/alert-bar.html

Reply:
ah, it's alert type.